.github/workflows/test.yaml

```diff
@@ -0,0 +1,61 @@
+name: Test
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      IGN_DB_USERNAME: gazebo
+      IGN_DB_PASSWORD: 1234
+      IGN_DB_NAME: gazebo_db
+      IGN_DB_MAX_OPEN_CONNS: 66
+    services:
+      mysql:
+        image: mysql:5.7
+        ports:
+          - 3306
+        env:
+          MYSQL_USER: ${{ env.IGN_DB_USERNAME }}
+          MYSQL_PASSWORD: ${{ env.IGN_DB_PASSWORD }}
+          MYSQL_DATABASE: "${{ env.IGN_DB_NAME }}_test"
+          MYSQL_RANDOM_ROOT_PASSWORD: true
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
```

Comment:
Not for this PR, but does it make sense to extract this to a `gazebo-web/.github` workflow you can call to setup a database? Something like Fuel (or any other project using a DB) would benefit from this.

Reply:
Thanks for the question. I really like the idea of applying DRY, but sadly, the `mysql` service is coupled to this workflow, I don't think you can extract it to a different one.

Reply:
Can we extract it nonetheless to have our CI resources available in a single place, even if it is for copy and pasting?

Reply:
I've been looking at https://github.com/gazebo-web/gz-go/pull/8/files, and it's running a job that's exactly the same as this one. Is it not possible to create a `test` workflow in the `gazebo-web/.github` repo that can then be called from here using `uses`?

Something like

```actions
jobs:
  test:
    uses: gazebo-web/.github/.github/workflows/go-test.yaml@<version>
```

Reply:
Friendly follow up on this: @AlejoAsd

Reply:
Ah, I was not aware that the starter template included a database job. This is exactly what I intended with my previous comment. Thanks a lot!